Libraries/LibWebView/InspectorClient.cpp

```diff
@@ -633,7 +633,10 @@ String InspectorClient::generate_dom_tree(JsonObject const& dom_tree)
             if (name.equals_ignoring_ascii_case("BODY"sv) || name.equals_ignoring_ascii_case("FRAMESET"sv))
                 m_body_or_frameset_node_id = node_id;
 
-            auto tag = name.to_lowercase();
+            auto tag = name;
+            if (MUST(String::from_byte_string(node.get_byte_string("namespace"sv).value_or(""sv))) == "http://www.w3.org/1999/xhtml"sv) {
```

Comment:
There's a lot going on in this line - we don't need to allocate the String or do the `value_or` (and let's use the existing namespace constant):
```c++
if (node.get_byte_string("namespace"sv) == Web::Namespace::HTML.bytes_as_string_view())
    tag = name.to_lowercase();
```

Reply:
I tried using the namespace constant.
It does not work, because, for some reason, `Web::Namespace::HTML`, if included with `#include <LibWeb/Namespace.h>` in this file, ends up being the empty string.

Reply:
Ahh right, that's because they're initialized via `Web::Namespace::initialize_strings`, which isn't called from the UI process. That's an artifact from older times when we handled allocation failure, which we don't bother with for these strings anymore.

Reply:
Changed it now so that they're just regular constants.